Gemfile

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'bcrypt', '3.1.12'
+gem 'bootstrap-sass', '3.3.7'
+gem 'coffee-rails', '4.2.2'
+gem 'dotenv-rails'
+gem 'devise'
+gem 'jbuilder', '2.7.0'
+gem 'jquery-rails', '4.3.1'
+gem 'omniauth-facebook'
+gem 'puma', '3.9.1'
+gem 'rails', '5.1.6'
+gem 'sass-rails', '5.0.6'
+gem 'turbolinks', '5.0.1'
+gem 'uglifier', '3.2.0'
+
+group :development, :test do
+  gem 'byebug', '9.0.6', platform: :mri
+  gem 'rspec-rails'
+  gem 'sqlite3', '1.3.13'
```

Comment:
Let's use `pg` instead of `sqlite3`. :smile: 
See screenshots below:
   ![image](https://user-images.githubusercontent.com/36057474/70843770-34240880-1e38-11ea-89f2-9604ef63690d.png)

:bulb:  I would recommend re-creating a new rails app with this flag: `--database=postgresql`
:bulb: Using dot `.` instead of the name of the app will use the existing folder to create a new rails app: `rails new . --database=postgresql`
:bulb: You can use this command to both delete what is in the folder and install a new rails app: `git rm -r . -f && rails new . --database=postgresql`

Reply:
Fixed, installed project with Postgresql from beggining

Reply:
Hi, as previously mentioned you are to use PostgreSQL from the beginning not only for production.

Please address this to get the 👍

Reply:
Sorry, there was a bug with github desktop and didnt push the last commit which included the new gemfile. the branch is up to date and the changes can be seen